### DISCUSSION TITLE
[@types/chrome] Resource interface lacks the 'type' property

### DISCUSSION THREAD

**Discussion by moniuch** [DefinitelyTyped/DefinitelyTyped#67954] [Issues with a @types package]
The Resource interface lists only: `url`, `getContent()`, and `setContent()`. There is a fourth one `type` which is used by [this example](https://github.com/GoogleChrome/chrome-extensions-samples/blob/main/api-samples/devtools/inspectedWindow/panel.js) and is in fact being logged out during runtime (Chrome 120):

![image](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/681956/ebe6943b-ea5b-4b76-9590-09954a66541d)

[a comment by an automated account is not shown]

**Comment by nicolas377** [comment]
Hey! Thanks for pointing this out, I'm guessing you're referring to `chrome.devtools.inspectedWindow`, documented [here](https://developer.chrome.com/docs/extensions/reference/api/devtools/inspectedWindow#type-Resource). It looks like the property `type` isn't documented. DefinitelyTyped does include undocumented behavior in some circumstances, but personally, I don't think chrome is gonna be the best place to have something undocumented, since the docs are so comprehensive, and that feature could break across different versions (and different chromium browsers like edge). Feel free to open up a PR though.
